runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java

```diff
@@ -180,16 +197,18 @@ private void addWidgets()
 			icon.setSpriteId(SpriteID.WIKI_SELECTED);
 			client.setAllWidgetsAreOpTargetable(true);
 		});
-		icon.setAction(5, "Search"); // Start at option 5 so the target op is ontop
+
+		// set position to 5 if we want Lookup on left-click, or set position to 4 if we want Search
+		final int position = config.leftClickSearch() ? 4 : 5;
```

Comment:
`position` feels a bit broad and could have a better name. Isn't it actually the index of the search op?

Reply:
It's the index but I don't personally understand the order (if there's only 3 elements in the array... why is position 4 on top 😄)

Reply:
there are 10 slots in the array, and the optarget is as if it is in slot 4½, so you have to use something <=4 to put it before, and >=5 to put it after

Reply:
That makes sense. Thanks!